src/config.hpp.in

```diff
@@ -46,6 +46,7 @@
 #cmakedefine ENABLE_CNMEM
 #cmakedefine ENABLE_CUDA
 #cmakedefine ENABLE_IMPLICIT_CONVERSIONS
-#cmakedefine ENABLE_THIN_UM
+#cmakedefine DISABLE_RM
```

Comment:
It's probably a good idea to prefix these with CHAI_ so they don't name-conflict with other libraries. You can still have cmake set them up without renaming all the cmake config options using something like #define CHAI_DISABLE_RM = \@DISABLE_RM\@

Reply:
Okay, good suggestion, thanks! I'll get this merged then will have another branch rename the defines.